src/playlist.js

```diff
@@ -485,6 +510,7 @@ export const estimateSegmentRequestTime = function(segmentDuration,
 
 Playlist.duration = duration;
```

Comment:
If not for the UNSAFE_LIVE_SEGMENTS piece above, I think you could do:

`
Playlist = {
  duration,
  seekable,
  safeLiveIndex,
  etc...
};
`

Which would be cleaner.